restful.go

```diff
@@ -110,20 +110,15 @@ func postRestfulQuery(
 	data *execResponse, err error) {
 
 	requestID := uuid.New().String()
-
 	data, err = sr.FuncPostQueryHelper(ctx, sr, params, headers, body, timeout, requestID)
 
-	// errors other than context timeout and cancel would be returned to upper layers
-	if err != context.Canceled && err != context.DeadlineExceeded {
```

Comment:
shouldn't we still have this line here? at least for this patch?

Reply:
Yep! Bad diff.